bin/diff_results

```diff
@@ -204,19 +204,26 @@ def diff(before_file, after_file, summary_filename):
         for category in CATEGORIES:
             cat_index = CATEGORIES.index(category)
             if do_intervals_differ(before_class_cis[key][cat_index], after_class_cis[cat_index]):
-                if (sample['detailed_classification']['no steady state'] < base_case['detailed_classification']['no steady state'] and
-                    sample['detailed_classification']['slowdown'] < base_case['detailed_classification']['slowdown']):
+                if (sample['detailed_classification']['warmup'] + sample['detailed_classification']['flat'] >
+                        base_case['detailed_classification']['warmup'] + base_case['detailed_classification']['flat']):
                     summary[DIFF][vm][bench][CLASSIFICATIONS] = BETTER
                     break
-                elif (sample['detailed_classification']['no steady state'] >= base_case['detailed_classification']['no steady state'] and
-                    sample['detailed_classification']['slowdown'] >= base_case['detailed_classification']['slowdown']):
+                elif (sample['detailed_classification']['no steady state'] + sample['detailed_classification']['slowdown'] >
+                        base_case['detailed_classification']['slowdown'] >= base_case['detailed_classification']['slowdown']):
```

Comment:
I'm not sure about the `>=` here. Shouldn't it be `>`. If they're `==`, then (although things are bad) they are no worse than before?

Reply:
Ach, this typo is worse than you think, let me push a fix.